python/cudf/cudf/core/dataframe.py

```diff
@@ -5892,7 +5900,7 @@ def select_dtypes(self, include=None, exclude=None):
         3  False  2.0
         4   True  1.0
         5  False  2.0
-        """
+        """  # noqa: E501
```

Comment:
These noqas don't just apply to the current line, but to the entire docstring since flake8 won't parse these comments inside docstrings. This line isn't too long, but I assume some of the lines in the docstring are.

Reply:
Correct. The docstring has a line that is too long somewhere above this location. Typically this is from a URL or example code snippet that cannot be wrapped.

Reply:
Thanks!